discover/deployment/articles/06-upgrading-to-liferay-7-1/99-deprecated-apps.markdown

```diff
@@ -80,3 +80,11 @@ they're still available.
 ## Web Experience
 
 ## Forms and Workflow
+
+## Security 
+
+| App | &nbsp;Availability | &nbsp;Notes |
+| --- | ------------------ | ----------- |
+| OAuth 1.0a | Marketplace | Replaced by OAuth 2.0, which is included in the bundle. |
+| OpenID | Release (bundle) | Low usage. Use OAuth 2.0 instead. |
```

Comment:
Should be:
{{+| OpenID | Release (bundle) | Low usage. Use OpenID Connect instead. |}}

Reply:
Thanks!